[Bug] LDH module yarn resourcemanager start failed

### Search before asking

- [X] I searched the [issues](https://github.com/apache/linkis/issues) and found no similar issues.


### Linkis Component

linkis-dist

### Steps to reproduce

```
bash ./linkis-dist/helm/scripts/create-kind-cluster.sh \
    && bash ./linkis-dist/helm/scripts/install-mysql.sh true\
    && bash ./linkis-dist/helm/scripts/install-ldh.sh true \
    && bash ./linkis-dist/helm/scripts/install-charts-with-ldh.sh linkis linkis-demo true
```
ldh pod  process 
```
[root@ldh-68c9c5996b-gp5hb hadoop]# jps
144 NameNode
450 NodeManager
243 DataNode
1573 FlinkYarnSessionCli
1749 Jps
1180 HistoryServer
1148 RunJar
1149 RunJar
```

### Anything else

```
lrwxrwxrwx 1 root root 36 May 31 23:50 flink -> /opt/ldh/1.4.0-SNAPSHOT/flink-1.12.2
lrwxrwxrwx 1 root root 36 May 31 23:50 hadoop -> /opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4
lrwxrwxrwx 1 root root 45 May 31 23:50 hive -> /opt/ldh/1.4.0-SNAPSHOT/apache-hive-3.1.3-bin
lrwxrwxrwx 1 root root 49 May 31 23:50 spark -> /opt/ldh/1.4.0-SNAPSHOT/spark-3.2.1-bin-hadoop3.2
lrwxrwxrwx 1 root root 50 May 31 23:50 zookeeper -> /opt/ldh/1.4.0-SNAPSHOT/apache-zookeeper-3.5.9-bin
```

resouecemanager log 
```
2023-06-06 11:53:12,538 INFO  [main] resourcemanager.ResourceManager (LogAdapter.java:info(51)) - STARTUP_MSG: 
/************************************************************
STARTUP_MSG: Starting ResourceManager
STARTUP_MSG:   host = ldh-68c9c5996b-gp5hb/10.244.0.6
STARTUP_MSG:   args = []
STARTUP_MSG:   version = 3.3.4
STARTUP_MSG:   classpath = /opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/jsp-api-2.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/snappy-java-1.1.8.2.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/javax.servlet-api-3.1.0.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/listenablefuture-9999.0-empty-to-avoid-conflict-with-guava.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/jackson-jaxrs-1.9.13.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/reload4j-1.2.22.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/jsr311-api-1.1.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/commons-codec-1.15.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/nimbus-jose-jwt-9.8.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/kerby-pkix-1.0.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/protobuf-java-2.5.0.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/commons-lang3-3.12.0.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/jaxb-impl-2.2.3-1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/jackson-annotations-2.12.7.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/jersey-server-1.19.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/jetty-webapp-9.4.43.v20210629.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/commons-logging-1.1.3.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/kerby-xdr-1.0.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/hadoop-shaded-guava-1.1.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/zookeeper-jute-3.5.6.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/slf4j-reload4j-1.7.36.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/kerby-asn1-1.0.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/jetty-server-9.4.43.v20210629.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/kerb-server-1.0.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/kerb-crypto-1.0.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/curator-client-4.2.0.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/re2j-1.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/stax2-api-4.2.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/jersey-core-1.19.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/hadoop-auth-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/kerb-admin-1.0.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/j2objc-annotations-1.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/jackson-xc-1.9.13.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/curator-framework-4.2.0.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/hadoop-annotations-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/commons-collections-3.2.2.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/commons-cli-1.2.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/jackson-databind-2.12.7.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/jetty-servlet-9.4.43.v20210629.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/dnsjava-2.1.7.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/jetty-io-9.4.43.v20210629.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/kerb-core-1.0.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/gson-2.8.9.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/jetty-security-9.4.43.v20210629.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/audience-annotations-0.5.0.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/paranamer-2.3.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/jersey-json-1.19.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/asm-5.0.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/jsr305-3.0.2.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/curator-recipes-4.2.0.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/httpcore-4.4.13.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/jcip-annotations-1.0-1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/kerb-util-1.0.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/commons-daemon-1.0.13.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/metrics-core-3.2.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/jetty-util-ajax-9.4.43.v20210629.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/accessors-smart-2.4.7.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/commons-io-2.8.0.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/jetty-xml-9.4.43.v20210629.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/jul-to-slf4j-1.7.36.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/jakarta.activation-api-1.2.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/animal-sniffer-annotations-1.17.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/commons-math3-3.1.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/jettison-1.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/avro-1.7.7.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/failureaccess-1.0.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/kerb-common-1.0.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/kerby-util-1.0.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/jsch-0.1.55.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/commons-text-1.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/kerb-client-1.0.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/kerby-config-1.0.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/commons-compress-1.21.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/checker-qual-2.5.2.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/commons-net-3.6.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/zookeeper-3.5.6.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/netty-3.10.6.Final.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/jackson-mapper-asl-1.9.13.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/guava-27.0-jre.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/jackson-core-2.12.7.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/kerb-simplekdc-1.0.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/jaxb-api-2.2.11.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/slf4j-api-1.7.36.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/hadoop-shaded-protobuf_3_7-1.1.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/jackson-core-asl-1.9.13.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/httpclient-4.5.13.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/commons-beanutils-1.9.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/json-smart-2.4.7.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/kerb-identity-1.0.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/woodstox-core-5.3.0.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/token-provider-1.0.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/jetty-util-9.4.43.v20210629.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/commons-configuration2-2.1.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/jersey-servlet-1.19.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/lib/jetty-http-9.4.43.v20210629.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/hadoop-common-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/hadoop-kms-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/hadoop-nfs-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/hadoop-common-3.3.4-tests.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/common/hadoop-registry-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/netty-transport-sctp-4.1.77.Final.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/snappy-java-1.1.8.2.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/javax.servlet-api-3.1.0.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/netty-resolver-dns-native-macos-4.1.77.Final-osx-aarch_64.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/listenablefuture-9999.0-empty-to-avoid-conflict-with-guava.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/netty-transport-classes-kqueue-4.1.77.Final.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/jackson-jaxrs-1.9.13.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/reload4j-1.2.22.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/jsr311-api-1.1.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/commons-codec-1.15.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/nimbus-jose-jwt-9.8.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/kerby-pkix-1.0.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/protobuf-java-2.5.0.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/commons-lang3-3.12.0.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/jaxb-impl-2.2.3-1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/netty-handler-4.1.77.Final.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/jackson-annotations-2.12.7.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/jersey-server-1.19.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/jetty-webapp-9.4.43.v20210629.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/netty-resolver-dns-native-macos-4.1.77.Final-osx-x86_64.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/commons-logging-1.1.3.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/kerby-xdr-1.0.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/hadoop-shaded-guava-1.1.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/netty-codec-dns-4.1.77.Final.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/zookeeper-jute-3.5.6.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/okhttp-4.9.3.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/okio-2.8.0.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/kerby-asn1-1.0.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/jetty-server-9.4.43.v20210629.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/kerb-server-1.0.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/netty-codec-4.1.77.Final.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/kerb-crypto-1.0.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/netty-transport-4.1.77.Final.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/curator-client-4.2.0.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/re2j-1.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/stax2-api-4.2.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/jersey-core-1.19.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/hadoop-auth-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/kerb-admin-1.0.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/netty-codec-xml-4.1.77.Final.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/j2objc-annotations-1.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/jackson-xc-1.9.13.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/curator-framework-4.2.0.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/netty-transport-native-epoll-4.1.77.Final-linux-aarch_64.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/hadoop-annotations-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/commons-collections-3.2.2.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/commons-cli-1.2.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/netty-codec-http-4.1.77.Final.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/jackson-databind-2.12.7.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/netty-transport-native-kqueue-4.1.77.Final-osx-aarch_64.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/jetty-servlet-9.4.43.v20210629.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/dnsjava-2.1.7.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/netty-all-4.1.77.Final.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/jetty-io-9.4.43.v20210629.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/kerb-core-1.0.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/netty-buffer-4.1.77.Final.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/gson-2.8.9.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/jetty-security-9.4.43.v20210629.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/audience-annotations-0.5.0.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/paranamer-2.3.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/leveldbjni-all-1.8.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/jersey-json-1.19.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/asm-5.0.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/jsr305-3.0.2.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/netty-resolver-4.1.77.Final.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/curator-recipes-4.2.0.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/httpcore-4.4.13.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/jcip-annotations-1.0-1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/netty-codec-mqtt-4.1.77.Final.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/netty-codec-redis-4.1.77.Final.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/kerb-util-1.0.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/commons-daemon-1.0.13.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/netty-codec-smtp-4.1.77.Final.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/netty-transport-native-kqueue-4.1.77.Final-osx-x86_64.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/jetty-util-ajax-9.4.43.v20210629.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/kotlin-stdlib-1.4.10.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/accessors-smart-2.4.7.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/commons-io-2.8.0.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/jetty-xml-9.4.43.v20210629.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/netty-codec-stomp-4.1.77.Final.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/jakarta.activation-api-1.2.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/animal-sniffer-annotations-1.17.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/commons-math3-3.1.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/jettison-1.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/kotlin-stdlib-common-1.4.10.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/netty-handler-proxy-4.1.77.Final.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/avro-1.7.7.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/failureaccess-1.0.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/kerb-common-1.0.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/netty-common-4.1.77.Final.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/kerby-util-1.0.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/jsch-0.1.55.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/netty-codec-memcache-4.1.77.Final.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/commons-text-1.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/kerb-client-1.0.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/kerby-config-1.0.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/commons-compress-1.21.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/checker-qual-2.5.2.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/netty-resolver-dns-4.1.77.Final.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/netty-transport-native-epoll-4.1.77.Final-linux-x86_64.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/commons-net-3.6.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/zookeeper-3.5.6.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/netty-3.10.6.Final.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/jackson-mapper-asl-1.9.13.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/netty-transport-classes-epoll-4.1.77.Final.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/netty-transport-rxtx-4.1.77.Final.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/guava-27.0-jre.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/jackson-core-2.12.7.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/kerb-simplekdc-1.0.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/jaxb-api-2.2.11.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/hadoop-shaded-protobuf_3_7-1.1.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/jackson-core-asl-1.9.13.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/netty-resolver-dns-classes-macos-4.1.77.Final.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/netty-codec-http2-4.1.77.Final.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/netty-codec-haproxy-4.1.77.Final.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/httpclient-4.5.13.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/netty-transport-udt-4.1.77.Final.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/commons-beanutils-1.9.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/json-smart-2.4.7.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/kerb-identity-1.0.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/woodstox-core-5.3.0.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/netty-transport-native-unix-common-4.1.77.Final.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/token-provider-1.0.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/jetty-util-9.4.43.v20210629.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/commons-configuration2-2.1.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/jersey-servlet-1.19.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/json-simple-1.1.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/jetty-http-9.4.43.v20210629.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/lib/netty-codec-socks-4.1.77.Final.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/hadoop-hdfs-nfs-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/hadoop-hdfs-rbf-3.3.4-tests.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/hadoop-hdfs-client-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/hadoop-hdfs-native-client-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/hadoop-hdfs-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/hadoop-hdfs-rbf-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/hadoop-hdfs-native-client-3.3.4-tests.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/hadoop-hdfs-httpfs-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/hadoop-hdfs-3.3.4-tests.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/hdfs/hadoop-hdfs-client-3.3.4-tests.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/mapreduce/hadoop-mapreduce-client-app-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/mapreduce/hadoop-mapreduce-client-shuffle-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/mapreduce/hadoop-mapreduce-client-common-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/mapreduce/hadoop-mapreduce-client-hs-plugins-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/mapreduce/hadoop-mapreduce-client-core-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/mapreduce/hadoop-mapreduce-examples-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/mapreduce/hadoop-mapreduce-client-uploader-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/mapreduce/hadoop-mapreduce-client-hs-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/mapreduce/hadoop-mapreduce-client-jobclient-3.3.4-tests.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/mapreduce/hadoop-mapreduce-client-nativetask-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/mapreduce/hadoop-mapreduce-client-jobclient-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/lib/mssql-jdbc-6.2.1.jre7.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/lib/jna-5.2.0.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/lib/bcpkix-jdk15on-1.60.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/lib/objenesis-2.6.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/lib/javax-websocket-server-impl-9.4.43.v20210629.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/lib/json-io-2.5.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/lib/guice-servlet-4.0.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/lib/websocket-api-9.4.43.v20210629.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/lib/jakarta.xml.bind-api-2.3.2.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/lib/javax.websocket-api-1.0.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/lib/websocket-servlet-9.4.43.v20210629.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/lib/jline-3.9.0.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/lib/aopalliance-1.0.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/lib/asm-analysis-9.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/lib/snakeyaml-1.26.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/lib/asm-commons-9.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/lib/jackson-jaxrs-base-2.12.7.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/lib/websocket-server-9.4.43.v20210629.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/lib/bcprov-jdk15on-1.60.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/lib/jackson-jaxrs-json-provider-2.12.7.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/lib/websocket-common-9.4.43.v20210629.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/lib/javax.websocket-client-api-1.0.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/lib/metrics-core-3.2.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/lib/fst-2.50.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/lib/jersey-guice-1.19.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/lib/HikariCP-java7-2.4.12.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/lib/javax.inject-1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/lib/jersey-client-1.19.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/lib/swagger-annotations-1.5.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/lib/jetty-plus-9.4.43.v20210629.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/lib/jackson-module-jaxb-annotations-2.12.7.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/lib/jetty-jndi-9.4.43.v20210629.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/lib/geronimo-jcache_1.0_spec-1.0-alpha-1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/lib/java-util-1.9.0.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/lib/javax-websocket-client-impl-9.4.43.v20210629.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/lib/asm-tree-9.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/lib/jetty-annotations-9.4.43.v20210629.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/lib/websocket-client-9.4.43.v20210629.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/lib/ehcache-3.3.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/lib/jetty-client-9.4.43.v20210629.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/lib/guice-4.0.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/hadoop-yarn-services-core-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/hadoop-yarn-client-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/hadoop-yarn-server-router-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/hadoop-yarn-server-sharedcachemanager-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/hadoop-yarn-server-timeline-pluginstorage-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/hadoop-yarn-api-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/hadoop-yarn-server-tests-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/hadoop-yarn-applications-mawo-core-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/hadoop-yarn-applications-distributedshell-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/hadoop-yarn-applications-unmanaged-am-launcher-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/hadoop-yarn-server-applicationhistoryservice-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/hadoop-yarn-server-common-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/hadoop-yarn-common-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/hadoop-yarn-server-nodemanager-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/hadoop-yarn-registry-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/hadoop-yarn-server-web-proxy-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/hadoop-yarn-services-api-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/hadoop-yarn-server-resourcemanager-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/timelineservice/hadoop-yarn-server-timelineservice-hbase-client-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/timelineservice/hadoop-yarn-server-timelineservice-hbase-common-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/timelineservice/hadoop-yarn-server-timelineservice-documentstore-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/timelineservice/hadoop-yarn-server-timelineservice-hbase-coprocessor-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/timelineservice/hadoop-yarn-server-timelineservice-3.3.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/timelineservice/lib/hbase-protocol-1.4.8.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/timelineservice/lib/commons-digester-1.8.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/timelineservice/lib/rxjava-1.3.8.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/timelineservice/lib/joni-2.1.2.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/timelineservice/lib/rxjava-string-1.1.1.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/timelineservice/lib/rxjava-extras-0.8.0.17.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/timelineservice/lib/commons-validator-1.6.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/timelineservice/lib/azure-cosmosdb-commons-2.4.5.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/timelineservice/lib/java-uuid-generator-3.1.4.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/timelineservice/lib/hbase-common-1.4.8.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/timelineservice/lib/metrics-core-2.2.0.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/timelineservice/lib/azure-cosmosdb-gateway-2.4.5.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/timelineservice/lib/commons-csv-1.0.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/timelineservice/lib/commons-collections4-4.2.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/timelineservice/lib/rxnetty-0.4.20.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/timelineservice/lib/hbase-annotations-1.4.8.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/timelineservice/lib/hbase-client-1.4.8.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/timelineservice/lib/commons-lang-2.6.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/timelineservice/lib/jcodings-1.0.13.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/timelineservice/lib/azure-cosmosdb-2.4.5.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/timelineservice/lib/azure-cosmosdb-direct-2.4.5.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/share/hadoop/yarn/timelineservice/lib/htrace-core-3.1.0-incubating.jar:/opt/ldh/1.4.0-SNAPSHOT/hadoop-3.3.4/contrib/capacity-scheduler/*.jar
STARTUP_MSG:   build = https://github.com/apache/hadoop.git -r a585a73c3e02ac62350c136643a5e7f6095a3dbb; compiled by 'stevel' on 2022-07-29T12:32Z
STARTUP_MSG:   java = 1.8.0_332
************************************************************/
2023-06-06 11:53:12,548 INFO  [main] resourcemanager.ResourceManager (LogAdapter.java:info(51)) - registered UNIX signal handlers for [TERM, HUP, INT]
2023-06-06 11:53:12,921 INFO  [main] resourcemanager.RMNMInfo (RMInfo.java:register(55)) - Registered RMInfo MBean
2023-06-06 11:53:12,934 INFO  [main] conf.Configuration (Configuration.java:getConfResourceAsInputStream(2845)) - core-site.xml not found
2023-06-06 11:53:12,979 INFO  [main] conf.Configuration (Configuration.java:getConfResourceAsInputStream(2845)) - resource-types.xml not found
2023-06-06 11:53:12,980 INFO  [main] resource.ResourceUtils (ResourceUtils.java:addResourcesFileToConf(476)) - Unable to find 'resource-types.xml'.
2023-06-06 11:53:13,034 INFO  [main] conf.Configuration (Configuration.java:getConfResourceAsInputStream(2845)) - yarn-site.xml not found
2023-06-06 11:53:13,040 INFO  [main] metrics.GenericEventTypeMetrics (GenericEventTypeMetrics.java:registerMetrics(75)) - Registering GenericEventTypeMetrics
2023-06-06 11:53:13,044 INFO  [main] event.AsyncDispatcher (AsyncDispatcher.java:register(267)) - Registering class org.apache.hadoop.yarn.server.resourcemanager.RMFatalEventType for class org.apache.hadoop.yarn.server.resourcemanager.ResourceManager$RMFatalEventDispatcher
2023-06-06 11:53:13,082 INFO  [main] security.NMTokenSecretManagerInRM (NMTokenSecretManagerInRM.java:<init>(75)) - NMTokenKeyRollingInterval: 86400000ms and NMTokenKeyActivationDelay: 900000ms
2023-06-06 11:53:13,088 INFO  [main] security.RMContainerTokenSecretManager (RMContainerTokenSecretManager.java:<init>(81)) - ContainerTokenKeyRollingInterval: 86400000ms and ContainerTokenKeyActivationDelay: 900000ms
2023-06-06 11:53:13,094 INFO  [main] security.AMRMTokenSecretManager (AMRMTokenSecretManager.java:<init>(94)) - AMRMTokenKeyRollingInterval: 86400000ms and AMRMTokenKeyActivationDelay: 900000 ms
2023-06-06 11:53:13,146 INFO  [main] event.AsyncDispatcher (AsyncDispatcher.java:register(267)) - Registering class org.apache.hadoop.yarn.server.resourcemanager.recovery.RMStateStoreEventType for class org.apache.hadoop.yarn.server.resourcemanager.recovery.RMStateStore$ForwardingEventHandler
2023-06-06 11:53:13,150 INFO  [main] event.AsyncDispatcher (AsyncDispatcher.java:register(267)) - Registering class org.apache.hadoop.yarn.server.resourcemanager.NodesListManagerEventType for class org.apache.hadoop.yarn.server.resourcemanager.NodesListManager
2023-06-06 11:53:13,150 INFO  [main] resourcemanager.ResourceManager (ResourceManager.java:createScheduler(488)) - Using Scheduler: org.apache.hadoop.yarn.server.resourcemanager.scheduler.capacity.CapacityScheduler
2023-06-06 11:53:13,203 INFO  [main] event.AsyncDispatcher (AsyncDispatcher.java:register(267)) - Registering class org.apache.hadoop.yarn.server.resourcemanager.scheduler.event.SchedulerEventType for class org.apache.hadoop.yarn.server.resourcemanager.ResourceManager$SchedulerEventDispatcher
2023-06-06 11:53:13,204 INFO  [main] event.AsyncDispatcher (AsyncDispatcher.java:register(267)) - Registering class org.apache.hadoop.yarn.server.resourcemanager.rmapp.RMAppEventType for class org.apache.hadoop.yarn.server.resourcemanager.ResourceManager$ApplicationEventDispatcher
2023-06-06 11:53:13,206 INFO  [main] event.AsyncDispatcher (AsyncDispatcher.java:register(267)) - Registering class org.apache.hadoop.yarn.server.resourcemanager.rmapp.attempt.RMAppAttemptEventType for class org.apache.hadoop.yarn.server.resourcemanager.ResourceManager$ApplicationAttemptEventDispatcher
2023-06-06 11:53:13,207 INFO  [main] event.AsyncDispatcher (AsyncDispatcher.java:register(267)) - Registering class org.apache.hadoop.yarn.server.resourcemanager.rmnode.RMNodeEventType for class org.apache.hadoop.yarn.server.resourcemanager.ResourceManager$NodeEventDispatcher
2023-06-06 11:53:13,252 WARN  [main] impl.MetricsConfig (MetricsConfig.java:loadFirst(136)) - Cannot locate configuration: tried hadoop-metrics2-resourcemanager.properties,hadoop-metrics2.properties
2023-06-06 11:53:13,265 INFO  [main] impl.MetricsSystemImpl (MetricsSystemImpl.java:startTimer(378)) - Scheduled Metric snapshot period at 10 second(s).
2023-06-06 11:53:13,266 INFO  [main] impl.MetricsSystemImpl (MetricsSystemImpl.java:start(191)) - ResourceManager metrics system started
2023-06-06 11:53:13,297 INFO  [main] security.YarnAuthorizationProvider (YarnAuthorizationProvider.java:getInstance(59)) - org.apache.hadoop.yarn.security.ConfiguredYarnAuthorizer is instantiated.
2023-06-06 11:53:13,301 INFO  [main] event.AsyncDispatcher (AsyncDispatcher.java:register(267)) - Registering class org.apache.hadoop.yarn.server.resourcemanager.RMAppManagerEventType for class org.apache.hadoop.yarn.server.resourcemanager.RMAppManager
2023-06-06 11:53:13,309 INFO  [main] event.AsyncDispatcher (AsyncDispatcher.java:register(267)) - Registering class org.apache.hadoop.yarn.server.resourcemanager.amlauncher.AMLauncherEventType for class org.apache.hadoop.yarn.server.resourcemanager.amlauncher.ApplicationMasterLauncher
2023-06-06 11:53:13,664 INFO  [main] resourcemanager.RMNMInfo (RMNMInfo.java:<init>(66)) - Registered RMNMInfo MBean
2023-06-06 11:53:13,664 INFO  [main] monitor.RMAppLifetimeMonitor (RMAppLifetimeMonitor.java:serviceInit(67)) - Application lifelime monitor interval set to 3000 ms.
2023-06-06 11:53:13,669 INFO  [main] placement.MultiNodeSortingManager (MultiNodeSortingManager.java:serviceInit(59)) - Initializing NodeSortingService=MultiNodeSortingManager
2023-06-06 11:53:13,671 INFO  [main] util.HostsFileReader (HostsFileReader.java:refresh(239)) - Refreshing hosts (include/exclude) list
2023-06-06 11:53:13,677 INFO  [main] conf.Configuration (Configuration.java:getConfResourceAsInputStream(2845)) - capacity-scheduler.xml not found
2023-06-06 11:53:13,683 INFO  [main] scheduler.AbstractYarnScheduler (AbstractYarnScheduler.java:getMinimumAllocation(1524)) - Minimum allocation = <memory:1024, vCores:1>
2023-06-06 11:53:13,683 INFO  [main] scheduler.AbstractYarnScheduler (AbstractYarnScheduler.java:getMaximumAllocation(1536)) - Maximum allocation = <memory:8192, vCores:4>
2023-06-06 11:53:13,697 INFO  [main] service.AbstractService (AbstractService.java:noteFailure(267)) - Service org.apache.hadoop.yarn.server.resourcemanager.scheduler.capacity.CapacityScheduler failed in state INITED
org.apache.hadoop.yarn.exceptions.YarnException: Failed to initialize queues
	at org.apache.hadoop.yarn.server.resourcemanager.scheduler.capacity.CapacityScheduler.initializeQueues(CapacityScheduler.java:774)
	at org.apache.hadoop.yarn.server.resourcemanager.scheduler.capacity.CapacityScheduler.initScheduler(CapacityScheduler.java:346)
	at org.apache.hadoop.yarn.server.resourcemanager.scheduler.capacity.CapacityScheduler.serviceInit(CapacityScheduler.java:425)
	at org.apache.hadoop.service.AbstractService.init(AbstractService.java:164)
	at org.apache.hadoop.service.CompositeService.serviceInit(CompositeService.java:109)
	at org.apache.hadoop.yarn.server.resourcemanager.ResourceManager$RMActiveServices.serviceInit(ResourceManager.java:910)
	at org.apache.hadoop.service.AbstractService.init(AbstractService.java:164)
	at org.apache.hadoop.yarn.server.resourcemanager.ResourceManager.createAndInitActiveServices(ResourceManager.java:1410)
	at org.apache.hadoop.yarn.server.resourcemanager.ResourceManager.serviceInit(ResourceManager.java:344)
	at org.apache.hadoop.service.AbstractService.init(AbstractService.java:164)
	at org.apache.hadoop.yarn.server.resourcemanager.ResourceManager.main(ResourceManager.java:1698)
Caused by: java.lang.IllegalStateException: Queue configuration missing child queue names for root
	at org.apache.hadoop.yarn.server.resourcemanager.scheduler.capacity.CapacitySchedulerQueueManager.parseQueue(CapacitySchedulerQueueManager.java:234)
	at org.apache.hadoop.yarn.server.resourcemanager.scheduler.capacity.CapacitySchedulerQueueManager.initializeQueues(CapacitySchedulerQueueManager.java:162)
	at org.apache.hadoop.yarn.server.resourcemanager.scheduler.capacity.CapacityScheduler.initializeQueues(CapacityScheduler.java:765)
	... 10 more
2023-06-06 11:53:13,700 INFO  [main] service.AbstractService (AbstractService.java:noteFailure(267)) - Service RMActiveServices failed in state INITED
org.apache.hadoop.service.ServiceStateException: org.apache.hadoop.yarn.exceptions.YarnException: Failed to initialize queues
	at org.apache.hadoop.service.ServiceStateException.convert(ServiceStateException.java:105)
	at org.apache.hadoop.service.AbstractService.init(AbstractService.java:173)
	at org.apache.hadoop.service.CompositeService.serviceInit(CompositeService.java:109)
	at org.apache.hadoop.yarn.server.resourcemanager.ResourceManager$RMActiveServices.serviceInit(ResourceManager.java:910)
	at org.apache.hadoop.service.AbstractService.init(AbstractService.java:164)
	at org.apache.hadoop.yarn.server.resourcemanager.ResourceManager.createAndInitActiveServices(ResourceManager.java:1410)
	at org.apache.hadoop.yarn.server.resourcemanager.ResourceManager.serviceInit(ResourceManager.java:344)
	at org.apache.hadoop.service.AbstractService.init(AbstractService.java:164)
	at org.apache.hadoop.yarn.server.resourcemanager.ResourceManager.main(ResourceManager.java:1698)
Caused by: org.apache.hadoop.yarn.exceptions.YarnException: Failed to initialize queues
	at org.apache.hadoop.yarn.server.resourcemanager.scheduler.capacity.CapacityScheduler.initializeQueues(CapacityScheduler.java:774)
	at org.apache.hadoop.yarn.server.resourcemanager.scheduler.capacity.CapacityScheduler.initScheduler(CapacityScheduler.java:346)
	at org.apache.hadoop.yarn.server.resourcemanager.scheduler.capacity.CapacityScheduler.serviceInit(CapacityScheduler.java:425)
	at org.apache.hadoop.service.AbstractService.init(AbstractService.java:164)
	... 7 more
Caused by: java.lang.IllegalStateException: Queue configuration missing child queue names for root
	at org.apache.hadoop.yarn.server.resourcemanager.scheduler.capacity.CapacitySchedulerQueueManager.parseQueue(CapacitySchedulerQueueManager.java:234)
	at org.apache.hadoop.yarn.server.resourcemanager.scheduler.capacity.CapacitySchedulerQueueManager.initializeQueues(CapacitySchedulerQueueManager.java:162)
	at org.apache.hadoop.yarn.server.resourcemanager.scheduler.capacity.CapacityScheduler.initializeQueues(CapacityScheduler.java:765)
	... 10 more
2023-06-06 11:53:13,700 INFO  [main] impl.MetricsSystemImpl (MetricsSystemImpl.java:stop(210)) - Stopping ResourceManager metrics system...
2023-06-06 11:53:13,701 INFO  [main] impl.MetricsSystemImpl (MetricsSystemImpl.java:stop(216)) - ResourceManager metrics system stopped.
2023-06-06 11:53:13,701 INFO  [main] impl.MetricsSystemImpl (MetricsSystemImpl.java:shutdown(611)) - ResourceManager metrics system shutdown complete.
2023-06-06 11:53:13,701 INFO  [main] event.AsyncDispatcher (AsyncDispatcher.java:serviceStop(201)) - AsyncDispatcher is draining to stop, ignoring any new events.
2023-06-06 11:53:13,701 INFO  [main] service.AbstractService (AbstractService.java:noteFailure(267)) - Service ResourceManager failed in state INITED
org.apache.hadoop.service.ServiceStateException: org.apache.hadoop.yarn.exceptions.YarnException: Failed to initialize queues
	at org.apache.hadoop.service.ServiceStateException.convert(ServiceStateException.java:105)
	at org.apache.hadoop.service.AbstractService.init(AbstractService.java:173)
	at org.apache.hadoop.service.CompositeService.serviceInit(CompositeService.java:109)
	at org.apache.hadoop.yarn.server.resourcemanager.ResourceManager$RMActiveServices.serviceInit(ResourceManager.java:910)
	at org.apache.hadoop.service.AbstractService.init(AbstractService.java:164)
	at org.apache.hadoop.yarn.server.resourcemanager.ResourceManager.createAndInitActiveServices(ResourceManager.java:1410)
	at org.apache.hadoop.yarn.server.resourcemanager.ResourceManager.serviceInit(ResourceManager.java:344)
	at org.apache.hadoop.service.AbstractService.init(AbstractService.java:164)
	at org.apache.hadoop.yarn.server.resourcemanager.ResourceManager.main(ResourceManager.java:1698)
Caused by: org.apache.hadoop.yarn.exceptions.YarnException: Failed to initialize queues
	at org.apache.hadoop.yarn.server.resourcemanager.scheduler.capacity.CapacityScheduler.initializeQueues(CapacityScheduler.java:774)
	at org.apache.hadoop.yarn.server.resourcemanager.scheduler.capacity.CapacityScheduler.initScheduler(CapacityScheduler.java:346)
	at org.apache.hadoop.yarn.server.resourcemanager.scheduler.capacity.CapacityScheduler.serviceInit(CapacityScheduler.java:425)
	at org.apache.hadoop.service.AbstractService.init(AbstractService.java:164)
	... 7 more
Caused by: java.lang.IllegalStateException: Queue configuration missing child queue names for root
	at org.apache.hadoop.yarn.server.resourcemanager.scheduler.capacity.CapacitySchedulerQueueManager.parseQueue(CapacitySchedulerQueueManager.java:234)
	at org.apache.hadoop.yarn.server.resourcemanager.scheduler.capacity.CapacitySchedulerQueueManager.initializeQueues(CapacitySchedulerQueueManager.java:162)
	at org.apache.hadoop.yarn.server.resourcemanager.scheduler.capacity.CapacityScheduler.initializeQueues(CapacityScheduler.java:765)
	... 10 more
2023-06-06 11:53:13,702 INFO  [main] resourcemanager.ResourceManager (ResourceManager.java:transitionToStandby(1482)) - Transitioning to standby state
2023-06-06 11:53:13,702 INFO  [main] resourcemanager.ResourceManager (ResourceManager.java:transitionToStandby(1489)) - Transitioned to standby state
2023-06-06 11:53:13,702 ERROR [main] resourcemanager.ResourceManager (MarkerIgnoringBase.java:error(159)) - Error starting ResourceManager
org.apache.hadoop.service.ServiceStateException: org.apache.hadoop.yarn.exceptions.YarnException: Failed to initialize queues
	at org.apache.hadoop.service.ServiceStateException.convert(ServiceStateException.java:105)
	at org.apache.hadoop.service.AbstractService.init(AbstractService.java:173)
	at org.apache.hadoop.service.CompositeService.serviceInit(CompositeService.java:109)
	at org.apache.hadoop.yarn.server.resourcemanager.ResourceManager$RMActiveServices.serviceInit(ResourceManager.java:910)
	at org.apache.hadoop.service.AbstractService.init(AbstractService.java:164)
	at org.apache.hadoop.yarn.server.resourcemanager.ResourceManager.createAndInitActiveServices(ResourceManager.java:1410)
	at org.apache.hadoop.yarn.server.resourcemanager.ResourceManager.serviceInit(ResourceManager.java:344)
	at org.apache.hadoop.service.AbstractService.init(AbstractService.java:164)
	at org.apache.hadoop.yarn.server.resourcemanager.ResourceManager.main(ResourceManager.java:1698)
Caused by: org.apache.hadoop.yarn.exceptions.YarnException: Failed to initialize queues
	at org.apache.hadoop.yarn.server.resourcemanager.scheduler.capacity.CapacityScheduler.initializeQueues(CapacityScheduler.java:774)
	at org.apache.hadoop.yarn.server.resourcemanager.scheduler.capacity.CapacityScheduler.initScheduler(CapacityScheduler.java:346)
	at org.apache.hadoop.yarn.server.resourcemanager.scheduler.capacity.CapacityScheduler.serviceInit(CapacityScheduler.java:425)
	at org.apache.hadoop.service.AbstractService.init(AbstractService.java:164)
	... 7 more
Caused by: java.lang.IllegalStateException: Queue configuration missing child queue names for root
	at org.apache.hadoop.yarn.server.resourcemanager.scheduler.capacity.CapacitySchedulerQueueManager.parseQueue(CapacitySchedulerQueueManager.java:234)
	at org.apache.hadoop.yarn.server.resourcemanager.scheduler.capacity.CapacitySchedulerQueueManager.initializeQueues(CapacitySchedulerQueueManager.java:162)
	at org.apache.hadoop.yarn.server.resourcemanager.scheduler.capacity.CapacityScheduler.initializeQueues(CapacityScheduler.java:765)
	... 10 more
2023-06-06 11:53:13,705 INFO  [shutdown-hook-0] resourcemanager.ResourceManager (LogAdapter.java:info(51)) - SHUTDOWN_MSG: 
/************************************************************
SHUTDOWN_MSG: Shutting down ResourceManager at ldh-68c9c5996b-gp5hb/10.244.0.6
************************************************************/
core file size          (blocks, -c) unlimited
data seg size           (kbytes, -d) unlimited
scheduling priority             (-e) 0
file size               (blocks, -f) unlimited
pending signals                 (-i) 127944
max locked memory       (kbytes, -l) 64
max memory size         (kbytes, -m) unlimited
open files                      (-n) 1048576
pipe size            (512 bytes, -p) 8
POSIX message queues     (bytes, -q) 819200
real-time priority              (-r) 0
stack size              (kbytes, -s) 8192
cpu time               (seconds, -t) unlimited
max user processes              (-u) unlimited
virtual memory          (kbytes, -v) unlimited
file locks                      (-x) unlimited

```




### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

--- a/.github/workflows/integration-test.yml
+++ b/.github/workflows/integration-test.yml
@@ -152,6 +152,8 @@ jobs:
             sleep 20
           done
 
+          bash ./linkis-dist/helm/scripts/prepare-for-spark.sh
+
           #show linkis pod logs
           #POD_NAME=`kubectl get pods -n linkis -l app.kubernetes.io/instance=linkis-demo-cg-linkismanager -o jsonpath='{.items[0].metadata.name}'`
           #kubectl logs -n linkis  ${POD_NAME} -f --tail=10000
@@ -170,16 +172,15 @@ jobs:
 
           # Execute test by linkis-cli
           POD_NAME=`kubectl get pods -n linkis -l app.kubernetes.io/instance=linkis-demo-mg-gateway -o jsonpath='{.items[0].metadata.name}'`
-          kubectl exec -it -n linkis  ${POD_NAME} -- bash -c " \
+          kubectl exec -n linkis  ${POD_NAME} -- bash -c " \
           sh /opt/linkis/bin/linkis-cli -engineType shell-1 -codeType shell -code \"pwd\" ";
 
-          kubectl exec -it -n linkis  ${POD_NAME} -- bash -c " \
+          kubectl exec -n linkis  ${POD_NAME} -- bash -c " \
           sh /opt/linkis/bin/linkis-cli -engineType python-python2 -codeType python -code   'print(\"hello\")' "
 
-          #todo
-          #kubectl exec -it -n linkis  ${POD_NAME} -- bash -c " \
-          #sh /opt/linkis/bin/linkis-cli -engineType hive-3.1.3 -codeType hql -code   'show databases' "
+          kubectl exec -n linkis  ${POD_NAME} -- bash -c " \
+          sh /opt/linkis/bin/linkis-cli -engineType hive-3.1.3 -codeType hql -code   'show databases' "
 
-          #kubectl exec -it -n linkis  ${POD_NAME} -- bash -c " \
-          #sh /opt/linkis/bin/linkis-cli -engineType spark-3.2.1 -codeType sql -code   'show databases' "
+          kubectl exec -n linkis  ${POD_NAME} -- bash -c " \
+          sh /opt/linkis/bin/linkis-cli -engineType spark-3.2.1 -codeType sql -code   'show databases' "
         shell: bash

--- a/linkis-dist/bin/install-linkis-to-kubernetes.sh
+++ b/linkis-dist/bin/install-linkis-to-kubernetes.sh
@@ -86,6 +86,8 @@ tag(){
 make_linkis_image_with_mysql_jdbc(){
     ${ROOT_DIR}/docker/scripts/make-linkis-image-with-mysql-jdbc.sh
     docker tag linkis:with-jdbc linkis:dev
+    ${ROOT_DIR}/docker/scripts/make-ldh-image-with-mysql-jdbc.sh
+    docker tag linkis-ldh:with-jdbc linkis-ldh:dev
 }
 #creating a kind cluster
 create_kind_cluster(){

--- a/linkis-dist/docker/ldh-with-mysql-jdbc.Dockerfile
+++ b/linkis-dist/docker/ldh-with-mysql-jdbc.Dockerfile
@@ -1,0 +1,29 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+ARG LINKIS_IMAGE=linkis-ldh:dev
+
+######################################################################
+# linkis-ldh image with mysql jdbc
+######################################################################
+FROM ${LINKIS_IMAGE}
+
+ARG LDH_HOME=/opt/ldh/current
+ARG MYSQL_JDBC_VERSION=8.0.28
+
+COPY mysql-connector-java-${MYSQL_JDBC_VERSION}.jar ${LDH_HOME}/hive/lib/
+COPY mysql-connector-java-${MYSQL_JDBC_VERSION}.jar ${LDH_HOME}/spark/lib/

--- a/linkis-dist/docker/ldh.Dockerfile
+++ b/linkis-dist/docker/ldh.Dockerfile
@@ -75,6 +75,10 @@ ADD ldh-tars/spark-${SPARK_VERSION}-bin-hadoop${SPARK_HADOOP_VERSION}.tgz /opt/l
 ADD ldh-tars/flink-${FLINK_VERSION}-bin-scala_2.11.tgz /opt/ldh/${LINKIS_VERSION}/
 ADD ldh-tars/apache-zookeeper-${ZOOKEEPER_VERSION}-bin.tar.gz /opt/ldh/${LINKIS_VERSION}/
 
+RUN ln -s /opt/ldh/${LINKIS_VERSION}/spark-${SPARK_VERSION}-bin-hadoop${SPARK_HADOOP_VERSION} /opt/ldh/current/spark \
+    && ln -s /opt/ldh/${LINKIS_VERSION}/hadoop-${HADOOP_VERSION} /opt/ldh/current/hadoop \
+    && ln -s /opt/ldh/${LINKIS_VERSION}/apache-hive-${HIVE_VERSION}-bin /opt/ldh/current/hive
+
 RUN mkdir -p /etc/ldh \
     && mkdir -p /var/log/hadoop && chmod 777 -R /var/log/hadoop \
     && mkdir -p /var/log/hive && chmod 777 -R /var/log/hive \
@@ -91,9 +95,10 @@ RUN mkdir -p /etc/ldh \
 #ADD ldh-tars/mysql-connector-java-${MYSQL_JDBC_VERSION}.jar /opt/ldh/current/hive/lib/
 #ADD ldh-tars/mysql-connector-java-${MYSQL_JDBC_VERSION}.jar /opt/ldh/current/spark/jars/
 
-ENV JAVA_HOME /etc/alternatives/jre
-ENV PATH /opt/ldh/current/hadoop/bin:/opt/ldh/current/hive/bin:/opt/ldh/current/spark/bin:/opt/ldh/current/flink/bin:/opt/ldh/current/zookeeper/bin:$PATH
+ENV JAVA_HOME=/etc/alternatives/jre
+ENV PATH=/opt/ldh/current/hadoop/bin:/opt/ldh/current/hive/bin:/opt/ldh/current/spark/bin:/opt/ldh/current/flink/bin:/opt/ldh/current/zookeeper/bin:$PATH
 ENV HADOOP_CONF_DIR=/etc/ldh/hadoop
+ENV YARN_CONF_DIR=/etc/ldh/hadoop
 ENV HIVE_CONF_DIR=/etc/ldh/hive
 ENV SPARK_CONF_DIR=/etc/ldh/spark
 ENV FLINK_CONF_DIR=/etc/ldh/flink

--- a/linkis-dist/docker/scripts/make-ldh-image-with-mysql-jdbc.sh
+++ b/linkis-dist/docker/scripts/make-ldh-image-with-mysql-jdbc.sh
@@ -1,0 +1,51 @@
+#!/bin/bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#
+
+WORK_DIR=`cd $(dirname $0); pwd -P`
+
+. ${WORK_DIR}/utils.sh
+
+IMAGE_NAME=${IMAGE_NAME:-linkis-ldh:with-jdbc}
+LINKIS_IMAGE=${LINKIS_IMAGE:-linkis-ldh:dev}
+LINKIS_HOME=${LINKIS_HOME:-/opt/ldh/current}
+MYSQL_JDBC_VERSION=${MYSQL_JDBC_VERSION:-8.0.28}
+MYSQL_JDBC_FILENAME=mysql-connector-java-${MYSQL_JDBC_VERSION}.jar
+MYSQL_JDBC_URL="https://repo1.maven.org/maven2/mysql/mysql-connector-java/${MYSQL_JDBC_VERSION}/${MYSQL_JDBC_FILENAME}"
+
+BUILD_DIR=`mktemp -d -t linkis-build-XXXXX`
+
+echo "#          build dir: ${BUILD_DIR}"
+echo "#         base image: ${LINKIS_IMAGE}"
+echo "# mysql jdbc version: ${MYSQL_JDBC_VERSION}"
+
+download ${MYSQL_JDBC_URL} ${MYSQL_JDBC_FILENAME} ${BUILD_DIR}
+
+echo "try to exec: docker build -f ${WORK_DIR}/../ldh-with-mysql-jdbc.Dockerfile \
+  -t ${IMAGE_NAME} \
+  --build-arg LINKIS_IMAGE=${LINKIS_IMAGE} \
+  --build-arg LINKIS_HOME=${LINKIS_HOME} \
+  --build-arg MYSQL_JDBC_VERSION=${MYSQL_JDBC_VERSION} \
+  ${BUILD_DIR}"
+
+docker build -f ${WORK_DIR}/../ldh-with-mysql-jdbc.Dockerfile \
+  -t ${IMAGE_NAME} \
+  --build-arg LINKIS_IMAGE=${LINKIS_IMAGE} \
+  --build-arg LINKIS_HOME=${LINKIS_HOME} \
+  --build-arg MYSQL_JDBC_VERSION=${MYSQL_JDBC_VERSION} \
+  ${BUILD_DIR}
+
+echo "# done, image: ${IMAGE_NAME}"

--- a/linkis-dist/helm/charts/linkis/templates/jobs.yaml
+++ b/linkis-dist/helm/charts/linkis/templates/jobs.yaml
@@ -28,7 +28,16 @@ spec:
           command:
             - /bin/bash
             - -ecx
-            - >-
+            - |-
+              sed -i 's#@YARN_RESTFUL_URL#{{ .Values.linkis.deps.yarn.restfulUrl }}#g' {{ .Values.linkis.locations.homeDir }}/db/linkis_dml.sql
+              sed -i 's#@HADOOP_VERSION#{{ .Values.linkis.deps.hadoop.version }}#g' {{ .Values.linkis.locations.homeDir }}/db/linkis_dml.sql
+              sed -i 's#@YARN_AUTH_ENABLE#{{ .Values.linkis.deps.yarn.authEnable }}#g' {{ .Values.linkis.locations.homeDir }}/db/linkis_dml.sql
+              sed -i 's#@YARN_AUTH_USER#{{ .Values.linkis.deps.yarn.authUser }}#g' {{ .Values.linkis.locations.homeDir }}/db/linkis_dml.sql
+              sed -i 's#@YARN_AUTH_PWD#{{ .Values.linkis.deps.yarn.authPassword }}#g' {{ .Values.linkis.locations.homeDir }}/db/linkis_dml.sql
+              sed -i 's#@YARN_KERBEROS_ENABLE#{{ .Values.linkis.deps.yarn.kerberosEnable }}#g' {{ .Values.linkis.locations.homeDir }}/db/linkis_dml.sql
+              sed -i 's#@YARN_PRINCIPAL_NAME#{{ .Values.linkis.deps.yarn.principal }}#g' {{ .Values.linkis.locations.homeDir }}/db/linkis_dml.sql
+              sed -i 's#@YARN_KEYTAB_PATH#{{ .Values.linkis.deps.yarn.keytab }}#g' {{ .Values.linkis.locations.homeDir }}/db/linkis_dml.sql
+              sed -i 's#@YARN_KRB5_PATH#{{ .Values.linkis.deps.yarn.krb5 }}#g' {{ .Values.linkis.locations.homeDir }}/db/linkis_dml.sql
               mysql -h{{ .Values.linkis.datasource.host }} -P{{ .Values.linkis.datasource.port }} -u{{ .Values.linkis.datasource.username }} -p{{ .Values.linkis.datasource.password }} --default-character-set=utf8 -e "CREATE DATABASE IF NOT EXISTS {{ .Values.linkis.datasource.database }} DEFAULT CHARSET utf8 COLLATE utf8_general_ci";
               mysql -h{{ .Values.linkis.datasource.host }} -P{{ .Values.linkis.datasource.port }} -u{{ .Values.linkis.datasource.username }} -p{{ .Values.linkis.datasource.password }} -D{{ .Values.linkis.datasource.database }}  --default-character-set=utf8 -e "source {{ .Values.linkis.locations.homeDir }}/db//linkis_ddl.sql";
               mysql -h{{ .Values.linkis.datasource.host }} -P{{ .Values.linkis.datasource.port }} -u{{ .Values.linkis.datasource.username }} -p{{ .Values.linkis.datasource.password }} -D{{ .Values.linkis.datasource.database }}  --default-character-set=utf8 -e "source {{ .Values.linkis.locations.homeDir }}/db//linkis_dml.sql"

--- a/linkis-dist/helm/scripts/install-ldh.sh
+++ b/linkis-dist/helm/scripts/install-ldh.sh
@@ -16,7 +16,7 @@
 #
 
 WORK_DIR=`cd $(dirname $0); pwd -P`
-
+ROOT_DIR=${WORK_DIR}/../..
 . ${WORK_DIR}/common.sh
 
 set -e
@@ -27,6 +27,9 @@ echo "# LDH version: ${LINKIS_IMAGE_TAG}"
 
 # load image
 if [[ "X${USING_KIND}" == "Xtrue" ]]; then
+  echo "# Preparing LDH image ..."
+  ${ROOT_DIR}/docker/scripts/make-ldh-image-with-mysql-jdbc.sh
+  docker tag linkis-ldh:with-jdbc linkis-ldh:dev
   echo "# Loading LDH image ..."
   kind load docker-image linkis-ldh:${LINKIS_IMAGE_TAG} --name ${KIND_CLUSTER_NAME}
 fi

--- a/linkis-dist/helm/scripts/prepare-for-spark.sh
+++ b/linkis-dist/helm/scripts/prepare-for-spark.sh
@@ -21,22 +21,16 @@ WORK_DIR=`cd $(dirname $0); pwd -P`
 
 ## copy spark resource from ldh to linkis-cg-engineconnmanager
 
-LDH_POD_NAME=`kubectl get pods -n ldh -l app=ldh    -o jsonpath='{.items[0].metadata.name}'`
-kubectl cp -n ldh ${LDH_POD_NAME}:/opt/ldh/ ./ldh
-
+LDH_POD_NAME=`kubectl get pods -n ldh -l app=ldh -o jsonpath='{.items[0].metadata.name}'`
 ECM_POD_NAME=`kubectl get pods -n linkis -l app.kubernetes.io/instance=linkis-demo-cg-engineconnmanager -o jsonpath='{.items[0].metadata.name}'`
-kubectl cp ./ldh  -n linkis ${ECM_POD_NAME}:/opt/ ;
 
+kubectl exec -n ldh ${LDH_POD_NAME} -- tar -C /opt -cf - ldh | \
+kubectl exec -i -n linkis ${ECM_POD_NAME} -- tar -C /opt -xf - --no-same-owner
 
-kubectl exec -it -n linkis ${ECM_POD_NAME} -- bash -c "chmod +x /opt/ldh/1.3.0/spark-3.2.1-bin-hadoop3.2/bin/*"
-kubectl exec -it -n linkis ${ECM_POD_NAME} -- bash -c "ln -s /opt/ldh/1.3.0/spark-3.2.1-bin-hadoop3.2 /opt/ldh/current/spark"
-kubectl exec -it -n linkis ${ECM_POD_NAME} -- bash -c "ln -s /opt/ldh/1.3.0/hadoop-3.3.4 /opt/ldh/current/hadoop"
-kubectl exec -it -n linkis ${ECM_POD_NAME} -- bash -c "ln -s /opt/ldh/1.3.0/apache-hive-3.1.3-bin /opt/ldh/current/hive"
-
-
-kubectl exec -it -n linkis ${ECM_POD_NAME} -- bash -c "echo 'export SPARK_HOME=/opt/ldh/current/spark' |sudo tee --append /etc/profile"
-kubectl exec -it -n linkis ${ECM_POD_NAME} -- bash -c "echo 'export PATH=\$SPARK_HOME/bin:\$PATH' |sudo tee --append  /etc/profile"
-kubectl exec -it -n linkis ${ECM_POD_NAME} -- bash -c "source /etc/profile"
+kubectl exec -n linkis ${ECM_POD_NAME} -- bash -c "sudo mkdir -p /appcom/Install && sudo chmod 0777 /appcom/Install && ln -s /opt/ldh/current/spark /appcom/Install/spark"
+kubectl exec -n linkis ${ECM_POD_NAME} -- bash -c "echo 'export SPARK_HOME=/opt/ldh/current/spark' |sudo tee --append /etc/profile"
+kubectl exec -n linkis ${ECM_POD_NAME} -- bash -c "echo 'export PATH=\$SPARK_HOME/bin:\$PATH' |sudo tee --append  /etc/profile"
+kubectl exec -n linkis ${ECM_POD_NAME} -- bash -c "source /etc/profile"
 
 # add ecm dns for ldh pod
 ECM_POD_IP=`kubectl get pods -n linkis -l app.kubernetes.io/instance=linkis-demo-cg-engineconnmanager -o jsonpath='{.items[0].status.podIP}'`
@@ -45,7 +39,4 @@ ECM_POD_SUBDOMAIN=`kubectl get pods -n linkis -l app.kubernetes.io/instance=link
 
 ECM_DNS="${ECM_POD_IP}   ${ECM_POD_NAME}.${ECM_POD_SUBDOMAIN}.linkis.svc.cluster.local"
 
-kubectl exec -it -n ldh ${LDH_POD_NAME} -- bash -c "echo ${ECM_DNS} |sudo tee --append  /etc/hosts"
-
-
-rm -rf ldh;
+kubectl exec -n ldh ${LDH_POD_NAME} -- bash -c "echo ${ECM_DNS} |sudo tee --append  /etc/hosts"

--- a/linkis-dist/helm/scripts/resources/ldh/configmaps/configmap-hadoop.yaml
+++ b/linkis-dist/helm/scripts/resources/ldh/configmaps/configmap-hadoop.yaml
@@ -592,3 +592,126 @@ data:
         </queue>
       </queue>
     </allocations>
+  capacity-scheduler.xml: |
+    <!--
+      Licensed under the Apache License, Version 2.0 (the "License");
+      you may not use this file except in compliance with the License.
+      You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+      Unless required by applicable law or agreed to in writing, software
+      distributed under the License is distributed on an "AS IS" BASIS,
+      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+      See the License for the specific language governing permissions and
+      limitations under the License. See accompanying LICENSE file.
+    -->
+    <configuration>
+
+      <property>
+        <name>yarn.scheduler.capacity.maximum-applications</name>
+        <value>4</value>
+        <description>
+          Maximum number of applications that can be pending and running.
+        </description>
+      </property>
+
+      <property>
+        <name>yarn.scheduler.capacity.maximum-am-resource-percent</name>
+        <value>0.5</value>
+        <description>
+          Maximum percent of resources in the cluster which can be used to run
+          application masters i.e. controls number of concurrent running
+          applications.
+        </description>
+      </property>
+
+      <property>
+        <name>yarn.scheduler.capacity.resource-calculator</name>
+        <value>org.apache.hadoop.yarn.util.resource.DefaultResourceCalculator</value>
+        <description>
+          The ResourceCalculator implementation to be used to compare
+          Resources in the scheduler.
+          The default i.e. DefaultResourceCalculator only uses Memory while
+          DominantResourceCalculator uses dominant-resource to compare
+          multi-dimensional resources such as Memory, CPU etc.
+        </description>
+      </property>
+
+      <property>
+        <name>yarn.scheduler.capacity.root.queues</name>
+        <value>default</value>
+        <description>
+          The queues at the this level (root is the root queue).
+        </description>
+      </property>
+
+      <property>
+        <name>yarn.scheduler.capacity.root.default.capacity</name>
+        <value>100</value>
+        <description>Default queue target capacity.</description>
+      </property>
+
+      <property>
+        <name>yarn.scheduler.capacity.root.default.user-limit-factor</name>
+        <value>1</value>
+        <description>
+          Default queue user limit a percentage from 0.0 to 1.0.
+        </description>
+      </property>
+
+      <property>
+        <name>yarn.scheduler.capacity.root.default.maximum-capacity</name>
+        <value>100</value>
+        <description>
+          The maximum capacity of the default queue.
+        </description>
+      </property>
+
+      <property>
+        <name>yarn.scheduler.capacity.root.default.state</name>
+        <value>RUNNING</value>
+        <description>
+          The state of the default queue. State can be one of RUNNING or STOPPED.
+        </description>
+      </property>
+
+      <property>
+        <name>yarn.scheduler.capacity.root.default.acl_submit_applications</name>
+        <value>*</value>
+        <description>
+          The ACL of who can submit jobs to the default queue.
+        </description>
+      </property>
+
+      <property>
+        <name>yarn.scheduler.capacity.root.default.acl_administer_queue</name>
+        <value>*</value>
+        <description>
+          The ACL of who can administer jobs on the default queue.
+        </description>
+      </property>
+
+      <property>
+        <name>yarn.scheduler.capacity.node-locality-delay</name>
+        <value>40</value>
+        <description>
+          Number of missed scheduling opportunities after which the CapacityScheduler
+          attempts to schedule rack-local containers.
+          Typically this should be set to number of nodes in the cluster, By default is setting
+          approximately number of nodes in one rack which is 40.
+        </description>
+      </property>
+
+      <property>
+        <name>yarn.scheduler.capacity.queue-mappings-override.enable</name>
+        <value>false</value>
+        <description>
+          If a queue mapping is present, will it override the value specified
+          by the user? This can be used by administrators to place jobs in queues
+          that are different than the one specified by the user.
+          The default is false.
+        </description>
+      </property>
+
+    </configuration>

